### PR TITLE
 Toggle Switch button added to Navbar

### PR DESCRIPTION
**Pull request by hasuwini77**
Added a switch button to toggle dark and light mode [ just the button ] Not the actual function yet ;)

### CODE REVIEW

File: src/components/Navbar/index.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
-import {Navbar, NavbarBrand, NavbarContent, NavbarItem, Link, Button} from "@nextui-org/react";
+import { Navbar, NavbarBrand, NavbarContent, NavbarItem, Link, Button } from "@nextui-org/react";
 import { AcmeLogo } from "../../assets/AcmeLogo";
```
Comment (davidboll) [inline src/components/Navbar/index.jsx:3]: Is this AcmeLogo used anywhere? If not, you can delete it :)

**Review by davidboll — COMMENTED** [review]
Nicely done :) Just check my one comment

File: src/components/Navbar/index.jsx (hunk 1 above)
Comment (hasuwini77) [reply src/components/Navbar/index.jsx:3]: It's used in the Navbar it's the "logo". We thought it would be best to delete it when we have the new one :)

**Review by davidboll — APPROVED** [review]
Approved :)

**Pull request merged**